sdk/spring/azure-spring-boot/README.md

```diff
@@ -41,7 +41,8 @@ Alternatively you may want to use the [starters](https://github.com/Azure/azure-
 ## Key concepts
 This project provides auto-configuration for the following Azure services:
 
-- [Azure Active Directory](../azure-spring-boot-starter-active-directory)
+- [Spring Boot 2.2 for Azure Active Directory](../azure-spring-boot-2-2-starter-active-directory)
```

Comment:
I think `Azure Active Directory for Spring Boot 2.2` will make more sense. Same to other items.